src/wazuh_db/wdb.c

```diff
@@ -118,6 +118,7 @@ static const char *SQL_STMT[] = {
     [WDB_STMT_GLOBAL_LABELS_DEL] = "DELETE FROM labels WHERE id = ?;",
     [WDB_STMT_GLOBAL_LABELS_SET] = "INSERT INTO labels (id, key, value) VALUES (?,?,?);",
     [WDB_STMT_GLOBAL_UPDATE_AGENT_KEEPALIVE] = "UPDATE agent SET last_keepalive = CASE WHEN last_keepalive IS NULL THEN 0 ELSE STRFTIME('%s', 'NOW') END, sync_status = ? WHERE id = ?;",
+    [WDB_STMT_GLOBAL_UPDATE_AGENT_CONNECTION_STATUS] = "UPDATE agent SET connection_status = ? WHERE id = ?;",
```

Comment:
Would it be necessary to add a constraint in the query to avoid overriding the status of the manager? The ID may be checked before.

Reply:
I think we shouldn't. Keeping the possibility to change manager status makes it more flexible.

Reply:
FIXED

Reply:
Just keeping it as it is to have the flexibility of setting the manager status according to the @palaciosjeremias comment.